index.bs

```diff
@@ -87,20 +87,20 @@ interface VRDisplay : EventTarget {
   readonly attribute DOMString displayName;
 
   /**
+   * Populates the passed VRFrameData with the information required to render
+   * the current frame.
+   */
+  void getCurrentFrameData(VRFrameData frameData);
```

Comment:
Agree on both fronts. And I'm testing if responding to these mails will auto comment for me. :-)

> On Sep 7, 2016, at 10:04 PM, Brandon Jones <notifications@github.com> wrote:
> 
> In index.bs:
> 
> > @@ -87,20 +87,20 @@ interface VRDisplay : EventTarget {
> >    readonly attribute DOMString displayName;
> >  
> >    /**
> > +   * Populates the passed VRFrameData with the information required to render
> > +   * the current frame.
> > +   */
> > +  void getCurrentFrameData(VRFrameData frameData);
> The more code and docs I write about this the more this function name feels overly verbose to me. Anyone object to simply getFrameData()?
> 
> Also, I think it should return a bool to indicate whether or not the frame data was populated.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>